apps/nextjs-website/src/app/page.tsx

```diff
@@ -63,7 +70,13 @@ const Home = async () => {
           logoUrl: product.logo.url,
         }))}
       />
-      <NotSsrWebinarsSection webinars={[...webinars]} />
+
+      <NotSsrWebinarsSection
+        title={'dontLoseNext'}
+        webinars={[...futureWebinars]}
+      />
+      <PastWebinarsShowcase webinars={[...pastWebinars]} />
```

Comment:
This component should be `dynamic` too. I would consider it as part of the WebinarsSection (that is to suggest you to remove the component from the page template to the WebinarSection component)